docker-compose.prod.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+
+services:
+  rdf-differ:
+    container_name: rdf_differ
+    build:
+      context: .
+      dockerfile: Dockerfile.prod
```

Comment:
best is when we have a single Dockerfile and we can provide an env file to specify the variations